pkg/manifest/manifest.go

```diff
@@ -249,9 +249,9 @@ type SetupLogger struct {
 
 // LocalServer represents a list of mocked Viceroy resources.
 type LocalServer struct {
-	Backends     map[string]LocalBackend     `toml:"backends"`
-	Dictionaries map[string]LocalDictionary  `toml:"dictionaries,omitempty"`
-	ObjectStore  map[string]LocalObjectStore `toml:"object_stores,omitempty"`
+	Backends     map[string]LocalBackend       `toml:"backends"`
+	Dictionaries map[string]LocalDictionary    `toml:"dictionaries,omitempty"`
+	ObjectStore  map[string][]LocalObjectStore `toml:"object_store,omitempty"`
```

Comment:
@chrispoole643 were you documenting the ObjectStore in https://developer.fastly.com/reference/compute/fastly-toml/ ? I thought I recalled you doing so but couldn't see it in there at the moment. At any rate, we'll want to ensure that's kept consistent with what's here ☝🏻